recipes-bsp/optee-os/optee-os-iot2050_3.21.0.bb

```diff
@@ -11,10 +11,10 @@
 require recipes-bsp/optee-os/optee-os-custom.inc
 require optee-os-iot2050_${PV}.inc
 
-# StMM integration
-DEPENDS:append:secureboot = " edk2-standalonemm-rpmb"
-DEBIAN_BUILD_DEPENDS:append:secureboot = ", edk2-standalonemm-rpmb"
-OPTEE_EXTRA_BUILDARGS:append:secureboot = " \
+# StMM integration, required by UEFI auth variable management
+DEPENDS:append:uefi = " edk2-standalonemm-rpmb"
+DEBIAN_BUILD_DEPENDS:append:uefi = ", edk2-standalonemm-rpmb"
+OPTEE_EXTRA_BUILDARGS:append:uefi = " \
```

Comment:
`uefi` is a bit to generic for this. We may eventually switch to UEFI-only booting, but even then this override is only valid in secure UEFI mode. How about `uefi-stmm` or so?

Reply:
> but even then this override is only valid in secure UEFI mode

Couldn't we reuse this StMM for non-secure UEFI as well?

Reply:
How? The RPMB would not be protected and on some boards not even available?

Reply:
I was thinking of letting only advanced variants use it, but that needs separate building for basic and advanced. So no, StMM is a no go for generic UEFI.

Let's go with `uefi-stmm`.